scheduler/scheduler-server/src/test/java/functionaltests/db/schedulerdb/TestLoadJobsPagination.java

```diff
@@ -398,6 +422,29 @@ public void testPagingAndFiltering() throws Exception {
         Assert.assertEquals(DEFAULT_USER_NAME, jobInfo.getJobOwner());
         Assert.assertEquals(SUBMISSION_MODE_REST_API, jobInfo.getSubmissionMode());
 
+        jobs = dbManager.getJobs(0,
+                                 1,
+                                 null,
+                                 null,
+                                 false,
+                                 true,
+                                 true,
+                                 true,
+                                 false,
+                                 true,
+                                 null,
+                                 null,
+                                 null,
+                                 null,
+                                 null,
+                                 null,
+                                 sortParameters,
+                                 DateTime.now().minusHours(2).getMillis(),
+                                 DateTime.now().minusHours(1).getMillis())
+                        .getList();
+
+        Assert.assertEquals(jobs.size(), 0);
```

Comment:
here you only test an empty result.

I think you should add more tests (that returns or not results), using combinations of greaterThan, lessThan parameters (for example greaterThan only, lessThen only, both greater and lesser)